src/main/resources/doc/fr/html/libs/gates/controlled.html

```diff
@@ -0,0 +1,278 @@
+              </td>
+              <td class="evalue">
+                E
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </center>
+	  </p>
+      <p>
+        Les tampons contrôlés peuvent être utiles lorsque vous avez un fil (souvent appelé <em>bus</em>) dont la valeur doit correspondre à une des sorties de plusieurs composants. En plaçant un tampon contrôlé entre la sortie de chaque composant et le bus, vous pouvez contrôler si la sortie de ce composant est alimentée ou non sur le bus.
+      </p>
+      <h2>
+        Pins (en supposant que le composant est orienté vers l'est, la ligne de contrôle est à droite)
+      </h2>
+	  <div class="attliste">
+      <dl>
+        <dt class="lib">
+		  Bord ouest:
+        </dt>
+        <dd>
+          L'entrée de la composante qui sera utilisée pour déterminer la sortie si l'entrée de contrôle est à <b class="unov">1</b>. Le nombre de bits correspond à l'attribut largeur de données</b>.
+        </dd>
+        <dt class="lib">
+		  Bord sud:
+        </dt>
+        <dd>
+		  L'entrée de contrôle du composant. largeur de données 1.
+        </dd>
+        <dt class="lib">
+		  Bord est:
+        </dt>
+        <dd>
+		  La sortie du composant, qui sera flottante (<b class="uvalue">U</b>) si l'entrée de commande est <b class="zerov">0</b> ou flottante (<b class="uvalue">U</b>), la valeur d'erreur (<b class="evalue">E</b>) si l'entrée de commande est la valeur d'erreur (<b class="evalue">E</b>), et sera déterminée sur la base de l'entrée côté ouest si l'entrée de commande est à <b class="unov">1</b>. Le nombre de bits correspond à l'attribut largeur de données</b>.
+        </dd>
+      </dl>
+	  </div>
+      <h2>
+        Attributs
+      </h2>
+      <p>
+		Lorsque le composant est sélectionné ou en cours d'ajout, <b class="tkeybd">Alt-0</b> à <b class="tkeybd">Alt-9</b> modifient son attribut largeur de données</b> et les touches fléchées modifient son attribut Orientation</b>.
+      </p>
+	  <div class="attliste">
+      <dl>
+        <dt class="lib">
+          Orientation</b>
+        </dt>
+        <dd>
+          La direction du composant, sa sortie par rapport à son entrée.
+        </dd>
+        <dt class="lib">
+          largeur de données</b>
+        </dt>
+        <dd>
+          La largeur de bit de l'entrées et la sorties du composant.
+        </dd>
+        <dt class="lib">
+          Dimension dessins</b>
+        </dt>
+        <dd>
+          (Tampon controlé inverseur seulement) Détermine s'il faut dessiner une version plus grande ou plus petite du composant.
+        </dd>
+        <dt class="lib">
+          Position de la commande
+        </dt>
+        <dd>
+   		   L'emplacement de la ligne de contrôle, en imaginant que nous regardons la sortie depuis l'entrée : Si le composant est orienté vers l'est et est règlé <q>A droite,</q> la ligne de contrôle est au sud ; mais s'il est règlé <q>A gauche,</q> la ligne de contrôle est au nord.
```

Comment:
Shouldn't it be "À droite" et "À gauche"?

Reply:
Indeed, the new French spelling requires it.
I correct it